src/css/components/panel.scss

```diff
@@ -127,9 +140,9 @@ h4 + .panel {
   form {
     background-color: $color-background-light;
     flex-grow: 1;
-    margin: 1rem -1rem -1rem;
-    padding: 1rem;
-    width: 100%;
+    margin: 0;
+    padding: $grid-2;
+    display: block;
```

Comment:
Removing negative margin as ti was causing items to overlap in the layout. Utilizing grid variables.